std_msgs/Header header

# The name of this bone 
uint8 type

# The base of the bone, closest to the wrist. 
geometry_msgs/Point bone_start

# The end of the bone, closest to the finger tip. 
geometry_msgs/Point bone_end